finans/kassekladde.php

```diff
@@ -1627,7 +1626,7 @@ function saveScrollPosition() {
 		} elseif ($kontrolkonto && $kontrolsaldo) {
 			print "<td align=right>" . dkdecimal($kontrolsaldo, 2) . "</td>\n";
 		}
-		if ($kontrolkonto && abs($saldo[$y]) > 0) {
+		if ($kontrolkonto && abs((float)$saldo[$y]) > 0) {
```

Comment:
`$saldo` is most often filled with NULL values, which cannot be passed to `abs()` as of PHP 8.1. Therefore casted to float which removes the warning